pxtarget.json

```diff
@@ -43,71 +43,101 @@
+        "libs/azureiot",
         "libs/base",
+        "libs/buttons",
+        "libs/color",
+        "libs/color-coded-tilemap",
+        "libs/controller",
+        "libs/controller---none",
         "libs/core",
+        "libs/core---linux",
+        "libs/core---nrf52",
         "libs/core---samd",
         "libs/core---stm32",
-        "libs/core---nrf52",
-        "libs/core---linux",
         "libs/core---vm",
-        "libs/serial",
-        "libs/serial---linux",
         "libs/corgio",
+        "libs/d5prj",
         "libs/darts",
-        "libs/keyboard",
-        "libs/mouse",
-        "libs/microphone",
+        "libs/device",
+        "libs/edge-connector",
+        "libs/esp32",
+        "libs/f4prj",
+        "libs/feather",
+        "libs/game",
+        "libs/game---light",
         "libs/hw",
-        "libs/hw---stm32f401",
-        "libs/hw---samd51",
         "libs/hw---n3",
         "libs/hw---n4",
         "libs/hw---rpi",
+        "libs/hw---samd51",
+        "libs/hw---stm32f401",
         "libs/hw---vm",
-        "libs/feather",
-        "libs/sevenseg",
-        "libs/settings",
-        "libs/settings---files",
-        "libs/screen",
-        "libs/screen---st7735",
-        "libs/screen---linux",
-        "libs/screen---ext",
-        "libs/game",
-        "libs/game---light",
-        "libs/buttons",
-        "libs/mixer",
-        "libs/servo",
+        "libs/keyboard",
         "libs/light",
+        "libs/lightsensor",
+        "libs/microphone",
+        "libs/mixer",
+        "libs/mixer---ext",
+        "libs/mixer---linux",
+        "libs/mixer---none",
+        "libs/mixer---nrf52",
         "libs/mixer---samd",
         "libs/mixer---stm32",
-        "libs/mixer---nrf52",
-        "libs/mixer---none",
-        "libs/mixer---linux",
-        "libs/mixer---ext",
-        "libs/device",
-        "libs/power",
-        "libs/controller",
-        "libs/controller---none",
-        "libs/edge-connector",
-        "libs/esp32",
+        "libs/mouse",
         "libs/mqtt",
         "libs/net",
         "libs/net-game",
-        "libs/azureiot",
-        "libs/color",
         "libs/palette",
-        "libs/storyboard",
-        "libs/rotary-encoder",
+        "libs/power",
         "libs/radio",
         "libs/radio-broadcast",
-        "libs/d5prj",
-        "libs/f4prj",
+        "libs/rotary-encoder",
         "libs/rpiprj",
-        "libs/color-coded-tilemap",
-        "libs/template"
+        "libs/screen",
+        "libs/screen---ext",
+        "libs/screen---linux",
+        "libs/screen---st7735",
+        "libs/serial",
+        "libs/serial---linux",
+        "libs/servo",
+        "libs/settings",
+        "libs/settings---files",
+        "libs/sevenseg",
+        "libs/storyboard",
+        "libs/template",
+        "libs/thermometer"
     ],
+    "staticpkgdirs": {
+        "base": [
+            "libs/hw",
+            "libs/hw---n3",
+            "libs/hw---n4",
+            "libs/hw---rpi",
```

Comment:
Can we skip RPI or anything which is not popular?

Reply:
Could also remove hw---n4. I assume no board is using it today? Microbit is hw---n3 right? @mmoskal

Reply:
You probably don’t have to build cpp extensions for rpi but I think people on forum are using it in general. I don’t think there is any hw in production with n4. N3 is indeed micro bit.